pkg/dashboard/objects/repos_test.go

```diff
@@ -19,7 +22,22 @@ func envOr(name, def string) string {
 
 func TestLoadRepo(t *testing.T) {
 
-	filePath := envOr("HELM_REPOSITORY_CONFIG", helmpath.ConfigPath("repositories.yaml"))
+	fileName := "repositories.yaml"
+	tmpDir := t.TempDir()
+
+	os.Setenv("HELM_CONFIG_HOME", tmpDir)
```

Comment:
This would be too intrusive. Instead, just set `settings.RepositoryConfig` value

Reply:
Sir can I create a `testdata` folder and keep the file in there ? So it will be easy to grab it from there

Reply:
Yes, you can create a folder.